db/src/kv/transaction.rs

```diff
@@ -54,6 +58,8 @@ pub enum Value {
 	TransactionMeta(TransactionMeta),
 	BlockNumber(u32),
 	Configuration(Bytes),
+	Bloom(Bytes),
```

Comment:
Is this (`Bloom`, `Empty`) a part of some future PR or an redundant artifact?

Reply:
Sorry - `Empty` is used, but `Bloom` is not

Reply:
Ah, right

I was going to add bloom as well, but decided to add later

Reply:
With those reorgs it is unclear whether bloom is going to be useful